### DISCUSSION TITLE
Maui and xamarin forms

### DISCUSSION THREAD

**Discussion by developer9969** [mobile-dev-inc/Maestro#116] [General]
Can this be used with Maui or Xamarin forms?

Thanks

**Comment by wollac11** [comment]
Yes, it works well with MAUI and Xamarin applications (at least on Android - I have not tested it on iOS yet but it should be fine). These frameworks create native views, so from Maestro's perspective, they appear the same as native apps.

**Comment by ArthurSav** [comment]
Hi, Maestro should generally work with all apps since it interacts with app(s) via accessibility apis from their respective platforms (iOS/Android). Some UI frameworks like Compose, Flutter etc.. might require additional configuration to detect elements but it generally just works.

With that said, we don't have any official documentation for supporting Maui yet.
